vta/python/vta/testing/simulator.py

```diff
@@ -55,5 +57,15 @@ def stats():
     x = tvm.get_global_func("vta.simulator.profiler_status")()
     return json.loads(x)
 
+def tsim_init(hw_lib):
+    """Init hardware library for TSIM"""
+    cur_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
+    vta_build_path = os.path.join(cur_path, "..", "..", "..", "build")
+    ext = ".dylib" if sys.platform == "darwin" else ".so"
```

Comment:
check if extension already exists

Reply:
How about [this](https://github.com/dmlc/tvm/pull/3258/commits/c7b7a32b37ff3dd7b7e073c25bfdf4221796274b)?